pscheduler-archiver-kafka/kafka/data-is-valid

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+#
+# Validate data for kafka
+#
+
+import pscheduler
+
+try:
+    json = pscheduler.json_load(max_schema=1)
+except ValueError as ex:
+    pscheduler.succeed_json({
+        "valid": False,
+        "error": str(ex)
+        })
+
+data_validator = {
+    #Data validation json object
+    "type": "object",
+    "properties": {
+        "topic": {"$ref": "#/pScheduler/String"},
+        "server-address": {"$ref": "#/pScheduler/String"}
```

Comment:
This should be shortened to just `server`. 

Also take a look at what other parameters are available in `KafkaProducer` that might be worth passing allowing the user to pass through.  The client identifier and acks look useful.  At some point in the future, we'll probably have to get into authentication, but not this round.